tests/test_ir/test_utils.py

```diff
@@ -22,6 +24,33 @@
 from clp_ffi_py.wildcard_query import WildcardQuery
 
 
+class JsonLinesFileReader:
+    """
+    Class for reading JSON files.
+
+    It assumes each line in the file is a JSON string, and the parser parses each line into a JSON
+    object, and return then through a generator.
+    """
+
+    def __init__(self, file_path: Path):
+        """
+        Initializes the `JSONFileReader` with the given file path.
+
+        :param file_path: Path to the JSON file to read.
+        """
+        self.file_path: Path = file_path
+
+    def read_lines(self) -> Generator[Any, None, None]:
+        """
+        Reads each line in the JSON file, parses it as a JSON object, and yields the JSON object.
+
+        :yield: Parsed JSON object for each line in the file.
```

Comment:
Is `yield` a recognized docstring tag?

Reply:
Yes. We have `:yield:` in `clp_ffi_py.ir.readers`, and here's the generated doc with `yield`: https://docs.yscope.com/clp-ffi-py/main/api/clp_ffi_py.ir.html#clp_ffi_py.ir.ClpIrStreamReader.search